app/models/notification.rb

```diff
@@ -79,7 +84,36 @@ def unarchive_if_updated
   def update_from_api_response(api_response, unarchive: false)
     attrs = Notification.attributes_from_api_response(api_response)
     self.attributes = attrs
+    update_subject
```

Comment:
You're right, I forgot the feature flag here, added it to the `#update_subject` method